share/github-backup-utils/ghe-backup-config

```diff
@@ -46,7 +46,7 @@ GHE_HOSTNAME_PRESERVE="$GHE_HOSTNAME"
 # Source in the backup config file from the local working copy location first
 # and then falling back to the system location.
 config_found=false
-for f in "$GHE_BACKUP_CONFIG" "/etc/github-backup-utils/backup.config"; do
+for f in "$GHE_BACKUP_CONFIG" "$HOME/github-backup-utils/backup.config" "/etc/github-backup-utils/backup.config"; do
```

Comment:
In my experience, it'd be fairly standard behaviour for a config file in a user's home directory to override the settings in a system config file. Given that, would it be an idea to switch the order of the `$HOME/github-backup-utils/backup.config` and `/etc/github-backup-utils/backup.config` files in this loop?